src/components/forms/newListingForm.jsx

```diff
@@ -115,6 +121,33 @@ class NewListingForm extends Component{
 
         );
     }
+    
+    submitForm = () => {
+        console.log("made it to submitform")
+        this.handleSubmit(this.state)
+        this.setState(this.initialState)
+    }
+
+    handleSubmit = character => {
+        this.makePostCall(character).then(callResult => {
+          if (callResult.status === 201) {
+            this.setState({ characters: [...this.state.characters, callResult.data] });
+          }
+        });
+    }
+    
+    makePostCall(character) {
+      return axios.post('http://localhost:5000/listings', character)
```

Comment:
Make sure the routes haven't changed in the backend, so just make sure to look at the backend file for changes in the future, but otherwise LGTM.